src/Main.elm

```diff
@@ -2,21 +2,64 @@ module Main exposing (main)
 
 import Browser
 import Data
+import Either exposing (Either(..))
 import Html exposing (Html)
 import Html.Attributes as Attrs
+import Html.Events as Evts
+import Json.Decode as Json
+import Json.Encode as Json
+import Scrive.Data.Address as CD
```

Comment:
This seems a weird choice for an alias.

Reply:
What we usually do in-house is `import Scrive.Data.Address as Address`.
I agree it's not ideal tho.